frontend/src/scss/breakpoints.scss

```diff
@@ -0,0 +1,2 @@
+$phone: 600px;
```

Comment:
where did You get these breakpoints from?

Reply:
I modeled them after: https://vuetifyjs.com/en/customization/breakpoints 
Do you @Bosmanfrx have better suggestion how to define breakpoints?

Reply:
I was just curious. ;) There is no one set of breakpoints. Maybe we could create map of all breakpoints here?